Index: notification-daemon/src/themes/standard/theme.c
===================================================================
--- notification-daemon/src/themes/standard/theme.c	(revision 3009)
+++ notification-daemon/src/themes/standard/theme.c	(working copy)
@@ -610,7 +610,6 @@ create_notification(UrlClickedCb url_cli
 							 GDK_WINDOW_TYPE_HINT_NOTIFICATION);
 	gtk_widget_add_events(win, GDK_BUTTON_PRESS_MASK | GDK_BUTTON_RELEASE_MASK);
 	gtk_widget_realize(win);
-	gtk_widget_set_size_request(win, WIDTH, -1);
 
 	g_object_set_data_full(G_OBJECT(win), "windata", windata,
 						   (GDestroyNotify)destroy_windata);
@@ -812,7 +811,7 @@ set_notification_text(GtkWindow *nw, con
 	update_content_hbox_visibility(windata);
 
 	gtk_widget_set_size_request(
-		((body != NULL && *body == '\0')
+		((body != NULL && *body != '\0')
 		 ? windata->body_label : windata->summary_label),
 		WIDTH - (IMAGE_SIZE + IMAGE_PADDING) - 10,
 		-1);
